pkgs/development/mobile/android-image-utils/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "android-image-utils";
+  version = "2012-08-08";
```

Comment:
This package is really old and the last commit is 8 years old. Also it probably needs updates for newer Android versions past 4.4. Is there any good reason why it still should be added to nixpkgs?

Reply:
I used it yesterday in an attempt to fix my phone, so I figured I'd PR it.

Reply:
It is still pretty old and probably needs patches in the future to work when packages it relies on get updated.

Reply:
It *is* pretty old, true.
It only depends on zlib and that seems to be a pretty a stable library, no?

On the other hand, this looks like a pretty niche package, and it has hardly any documentation, so that might be a reason not to ad it.

I only PR'd it cause I had built it since I needed it, so it was easy enough for me to offer it to the community and let you all decide if it should be in nixpkgs.

Reply:
As an alternative for such outdated packages, consider contributing to [NUR](https://github.com/nix-community/NUR).

Reply:
Indeed, since upstream is unmaintained, the NUR seems to be the most appropriate place.